models/articles.js

```diff
@@ -7,3 +7,9 @@ exports.selectArticleById = (id) => {
       return rows;
     });
 };
+
+exports.selectAllArticles = () => {
+  return db.query(`SELECT * FROM ARTICLES ;`).then(({ rows }) => {
```

Comment:
need to make sure the article has a comment count, and is in the order asked for in the ticket.